src/components/product-hero.vue

```diff
@@ -60,6 +60,15 @@ export default {
       default: undefined
     },
 
+    /**
+     * The format you want the image
+     */
+    format: {
```

Comment:
Is this used?

Reply:
It's included with the rest of the props on line 118